src/GstImbSink.cpp

```diff
@@ -469,12 +469,16 @@ static void imb_sink_get_property(GObject* object, guint propId, GValue* value,
 
         case PROP_CURRENT_POS:
         {
+            double position = 0.0;
```

Comment:
Any reason why we can't do:
```suggestion
            self->cur_pos = 0.;
```